kratos/python_scripts/set_initial_state_process.py

```diff
@@ -1,51 +1,62 @@
-# Importing the Kratos Library
+"""Process for setting an initial state (strain, stress, deformation radient)."""
 import KratosMultiphysics
 
 
 def Factory(settings, Model):
+    """Create process."""
     if not isinstance(settings, KratosMultiphysics.Parameters):
         raise Exception(
             "expected input shall be a Parameters object, encapsulating a json string"
         )
     return SetInitialStateProcess(Model, settings["Parameters"])
 
 
-# All the processes python processes should be derived from "Process"
+def param_to_str(param):
```

Comment:
```suggestion
def __ParamToStr(param):
```
[Style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide). I'd also use name mangling with this one.

Reply:
Done

Reply:
@rubenzorrilla I don´t know why, but with the mangled name, tests fail:

```
ERROR: test_execution (test_factory.InitialStateInelasticity2Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/marcelo/Projects/Kratos/applications/ConstitutiveLawsApplication/tests/test_factory.py", line 54, in setUp
    self.test.Initialize()
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 60, in Initialize
    super().Initialize()
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/analysis_stage.py", line 89, in Initialize
    self.__CreateListOfProcesses() # has to be done after importing and preparing the ModelPart
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/analysis_stage.py", line 342, in __CreateListOfProcesses
    self._list_of_processes        = self._CreateProcesses("processes", order_processes_initialization)
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 101, in _CreateProcesses
    list_of_processes = super()._CreateProcesses(parameter_name, initialization_order)
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/analysis_stage.py", line 317, in _CreateProcesses
    list_of_processes += factory.ConstructListOfProcesses(value) # Does this work? or should it be processes[name]
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/process_factory.py", line 36, in ConstructListOfProcesses
    p = python_module.Factory(item, self.Model)
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/set_initial_state_process.py", line 11, in Factory
    return SetInitialStateProcess(Model, settings["Parameters"])
  File "/home/marcelo/Projects/Kratos/bin/current/KratosMultiphysics/set_initial_state_process.py", line 104, in __init__
    m = __ParamToStr(settings["imposed_strain_multiplier"])
NameError: name '_SetInitialStateProcess__ParamToStr' is not defined
```

If I use `_ParamToStr` instead of `__ParamToStr`, test finish normally. (Same thing for `__StrToFunction`)